Clarify that NFS with Cobbler is not supported (bsc#1240666) (#3948)
uyuni-project/uyuni-docs: modules/installation-and-upgrade/pages/hardware-requirements.adoc

[IMPORTANT]
====

> Do not use NFS for storage. It is not supported.

Reviewer: ```suggestion
Do not use NFS for storage.
It is not supported.
```

[IMPORTANT]
====

> Do not use NFS for {cobbler} or {postgresql} storage.

Author: Just rephrased the sentence. I'm not mentioning "file labeling" as it might bring confusion about why is not supported.

> Do not use NFS for {cobbler} or {postgresql} storage.

Reviewer: Thanks for improvements and background info!  Additionally, I committed a small wording change.  If nobody vetoes, I'll merge it later today.

> Do not use NFS for {cobbler} or {postgresql} storage, neither NFS on SELinux enviroments.
> It is not supported.

Reviewer: ```suggestion
Do not use NFS for {cobbler} or {postgresql} storage, neither NFS on SELinux environments.
These scenarios are not supported.
```
